<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auth_form</class>
 <widget class="QDialog" name="auth_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Oturum Açma Paneli</string>
  </property>
  
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QWidget" name="logo_wid" native="true">
     
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,1">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="4" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QPushButton" name="btn_cancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>İPTAL</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QPushButton" name="btn_login">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          
          <property name="text">
           <string>GİRİŞ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
         <widget class="QLabel" name="lbl_welcome">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Grenada</family>
            <pointsize>17</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>HOŞGELDİNİZ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignBottom">
         <widget class="QLabel" name="lbl_login">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <pointsize>8</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Lütfen Oturum Açınız..</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
         <widget class="QLineEdit" name="txt_user">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>admin</string>
          </property>
          <property name="placeholderText">
           <string>Kullanıcı Adı ...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
         <widget class="QLineEdit" name="txt_pass">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>1234</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Şifre ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/context.qrc"/>
 </resources>
 <connections/>
</ui>
